ramp-frontend/ramp_frontend/tests/test_utils.py

```diff
@@ -46,18 +48,6 @@ def client_session(database_connection):
         Model.metadata.drop_all(db)
 
 
-@_fail_no_smtp_server
```

Comment:
This test cannot work anymore without having a view and some requests.
However, we can rely on the integration test that makes a similar check for each email sent.